### DISCUSSION TITLE
Sponsorship for typegraphql-prisma?

### DISCUSSION THREAD

**Discussion by ElliotAtHelsinki** [prisma/prisma#24849] [Q&A] [answered]
[typegraphql-prisma](https://github.com/MichalLytek/typegraphql-prisma#readme) is a community generator that automatically generates TypeScript models and resolvers based on the `prisma schema`. It is my favourite way to build GraphQL APIs using Prisma. However, the project sadly lost financial support from Prisma last year; see here: [#385](https://github.com/MichalLytek/typegraphql-prisma/issues/385). Right now, the project appears to have only one developer, @MichalLytek, and he seems to be only able to provide some bug fixes and compatibility updates every now and then. Even so, there were a few bugs that haven't been fixed for quite some time, which I assume is because the developer doesn't have much time to spend on the project without financial support. It really is a shame. Could the Prisma team perhaps reconsider supporting the project? Thanks!

**Comment by ludralph** [comment, marked as answer]
Hi @ElliotAtHelsinki  👋 

Thank you so much for reaching out and for your interest in using Prisma!

We love hearing from our community and appreciate your enthusiasm. We continuously evaluate our support of community projects and will definitely keep your request in mind. If it's technically feasible, we’ll look into supporting your project in the future.

Thanks again for being a part of the Prisma community!
